server/game/gamesteps/conflict/initiateconflictprompt.js

```diff
@@ -178,7 +177,21 @@ class InitiateConflictPrompt extends UiPrompt {
     }
 
     recalculateCovert() {
+        // remove any selected defenders with covert
+        _.each(this.selectedDefenders, card => {
```

Comment:
Maybe I'm not understanding what this is supposed to do, but wouldn't `.filter` achieve what you're aiming to do here?

Reply:
Probably?  I need to set `.covert = false` on the card and also remove the card from the `selectedDefenders` array.  There may be a more concise way of doing this - any ideas?